libs/botan/Makefile

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (C) 2019 OpenWrt.org
```

Comment:
https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md
Don't add OpenWrt.org as copyright holder unless you're affiliated

Reply:
fixed, thanks